<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWidget</class>
 <widget class="QWidget" name="HomeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="checkInButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>打卡日历</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="learnButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>开始学习</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reciteButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>开始背诵</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recordButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>错题本</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reviewButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>复习错题</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
